### PR TITLE
modify add_like_count

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

File: dataAccessObjects.py [hunk 1]
```diff
@@ -142,6 +142,9 @@ def getFileInfo(self,imgId):
         else:
             #TODO raise exception
             return None
+    def addLikeAccount(self):
```
Comment (Billy4195) [inline dataAccessObjects.py]: Do you mean`addLikeCount`? or `addLikeAccount`?

File: dataAccessObjects.py [hunk 2]
```diff
@@ -142,6 +142,9 @@ def getFileInfo(self,imgId):
...
+        sql = 'UPDATE image_data SET img_like_count=img_like_count+1 WHERE img_id="{target_id}"'.format(target_id=str(target_id))
```
Comment (Billy4195) [inline dataAccessObjects.py]: The variable name should use `lower camel case`
And you hasn't declared the `target_id` above this line.

File: dataAccessObjects.py [hunk 3]
```diff
@@ -153,6 +156,9 @@ def getFileInfo(self,textId):
         else:
             #TODO raise exception
             return None
+    def addLikeAccount(self):
+        sql = 'UPDATE text_data SET text_like_count=text_like_count+1 WHERE text_id="{target_id}"'.format(target_id=str(target_id))
```
Comment (Billy4195) [inline dataAccessObjects.py]: The variable name should use `lower camel case`
And you hasn't declared the `target_id` above this line.

**Comment by stvreumi** [comment]
OK, please help me review again. Thanks.

File: dataAccessObjects.py [hunk 4]
```diff
@@ -142,6 +142,9 @@ def getFileInfo(self,imgId):
         else:
             #TODO raise exception
             return None
```
Comment (Billy4195) [inline dataAccessObjects.py]: Please add newline between two functions

File: dataAccessObjects.py [hunk 5]
```diff
@@ -153,6 +156,9 @@ def getFileInfo(self,textId):
...
```
Comment (Billy4195) [inline dataAccessObjects.py]: Same as above

**Comment by stvreumi** [comment]
OK, please help me review the change, thanks.

**Pull request merged**